howto/change-python.md

```diff
@@ -0,0 +1,87 @@
+# Bump Python Version for DjangoGirls.org Website
+
+In order to bump up the Python version for the DjangoGirls.org website, you 
+need to have access to our PythonAnywhere account, and follow these instructions:
+
+[Login to PythonAnywhere](https://www.pythonanywhere.com/user/djangogirls2/consoles/). 
+* Credentials can be found in 1Password.
+
+* Check if the Python version you want to upgrade to is supported by the current
+system image used on PythonAnywhere. 
+You can do so by clicking on the pen icon on the Web Tab or by 
+[reading this](https://help.pythonanywhere.com/pages/ChangingSystemImage#available-python-versions-for-system-images).
+ If the version of Python is already supported by the current image, there is 
+ no need to change the system image. 
+ If the version is not supported by the current system image, then the system
+ image needs to be upgraded. 
+
+* Check which `node` version we are using opening the `Bash Console` and 
```

Comment:
"Node" here is probably the name of the software, not the CLI command. Also, should UI elements that aren't to be typed (nor are CLI output) also be set in monospace or should they be highlighted in some other way, e.g., bold?
```suggestion
* Check which Node version we are using opening the **Bash Console** and 
```